tests/basic/pure_test.go

```diff
@@ -1014,3 +1017,285 @@ var _ = Describe("{RestartPXWhileVolCreate}", func() {
 		defer EndTorpedoTest()
 	})
 })
+
+var _ = Describe("{ResizePVCToMaxLimit}", func() {
+
+	/*
+		PTX: https://portworx.atlassian.net/browse/PTX-20636
+		TestRail: https://portworx.testrail.net/index.php?/tests/view/72657582
+	*/
+
+	// Backend represents the cloud storage provider for volume provisioning
+	type Backend string
+
+	const (
+		BackendPure    Backend = "PURE"
+		BackendVSphere Backend = "VSPHERE"
+		BackendUnknown Backend = "UNKNOWN"
+	)
+
+	// VolumeType represents the type of provisioned volume
+	type VolumeType string
+
+	const (
+		VolumeFADA    VolumeType = "FADA"
+		VolumeFBDA    VolumeType = "FBDA"
+		VolumeFACD    VolumeType = "FACD"
+		VolumeVsCD    VolumeType = "VsCD"
+		VolumeUnknown VolumeType = "UNKNOWN"
+	)
+
+	var (
+		contexts            = make([]*scheduler.Context, 0)
+		backend             = BackendUnknown
+		volumeMap           = make(map[VolumeType][]*api.Volume)
+		volumeCtxMap        = make(map[string]*scheduler.Context)
+		steps        uint64 = 5
+	)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("ResizePVCToMaxLimit", "Validate PVC resize to max limit", nil, 72657582)
```

Comment:
Is this even a valid testrail ID?

Reply:
https://portworx.testrail.net/index.php?/tests/view/72657582

Reply:
This is incorrect. The link you have mentioned is from a test run not the actual test case. The correct link is this - https://portworx.testrail.net/index.php?/cases/view/87940

Reply:
Done. Thank you.